### PR TITLE
PLT-477 - Profile tags rendering for Personalised Content

### PR DESCRIPTION
**Pull request by abdulrkalam**

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsresettlementpassportapi/service/ResettlementAssessmentService.kt [hunk 1]
```diff
@@ -489,13 +492,25 @@ class ResettlementAssessmentService(
...
+  }
+
+  fun processProfileTags(resettlementAssessmentEntity: ResettlementAssessmentEntity, pathway: Pathway): List<String> {
+    val tagList = mutableListOf<String>()
     resettlementAssessmentEntity.assessment.assessment.forEach {
-      if (it.questionId == TagAndQuestionMapping.NO_FIXED_ABODE.questionOptionId.toString()) {
-        tagList.add(TagAndQuestionMapping.NO_FIXED_ABODE)
-      } else if (it.questionId == TagAndQuestionMapping.HOME_ADAPTION_POST_RELEASE.questionOptionId.toString()) {
-        tagList.add(TagAndQuestionMapping.HOME_ADAPTION_POST_RELEASE)
+      log.info("Answer value for question id ${it.questionId} is ${it.answer.answer}")
```
Comment (tomhalson) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsresettlementpassportapi/service/ResettlementAssessmentService.kt:510]: I dont think we need to log each question and answer, some of the answers are considered sensitive and so we shouldnt put them in the logs

File: src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsresettlementpassportapi/data/Prisoners.kt [hunk 2]
```diff
@@ -67,4 +68,5 @@ data class Prisoner(
   val immediateNeedsSubmitted: Boolean,
   val preReleaseSubmitted: Boolean,
   val isInWatchlist: Boolean,
+  val profile: ProfileTagList,
```
Comment (mkerry-moj) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsresettlementpassportapi/data/Prisoners.kt:71]: I don't think we will need this in the search results, just on the get a single prisoner

Comment (abdulrkalam) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsresettlementpassportapi/data/Prisoners.kt:71]: yes this Prisoner data  is for singe prisoner. though the data class in Prisoners.kt, it is used for single prisoner only. The search for list prisoners using PrisonersSearch data class

File: src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsresettlementpassportapi/service/ResettlementAssessmentService.kt (hunk 1 above)
Comment (abdulrkalam) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsresettlementpassportapi/service/ResettlementAssessmentService.kt]: Added the log for local debug, forgot to remove it. it is done now!

File: src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsresettlementpassportapi/data/Prisoners.kt (hunk 2 above)
Comment (mkerry-moj) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsresettlementpassportapi/data/Prisoners.kt:71]: Oh it's just the way github had collapsed the diff, made it look like it was part of Prisoners class. Ignore :)

**Review by mkerry-moj — APPROVED** [review]

**Pull request merged**